Backend/NoCode.FlowerShop.Application/FlowerArrangements/Delete/DeleteFlowerArrangementValidator.cs

```diff
@@ -0,0 +1,10 @@
+﻿using FluentValidation;
+
+namespace NoCode.FlowerShop.Application.FlowerArrangements.Delete;
+public sealed class DeleteFlowerArrangementValidator : AbstractValidator<DeleteFlowerArrangementCommand>
+{
+    public DeleteFlowerArrangementValidator()
+    {
+        RuleFor(x => x.Id).NotEmpty();
```

Comment:
Not sure if ID can be empty (if that's a query parameter, then calling the endpoint without ID will probably result in HTTP 404 or 405 as it counts as an other endpoint, but I might be hallucinating)

Reply:
I think you are right - this check is excessive.